crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -761,9 +738,49 @@ async fn execute_operation(
 
     match child {
         Ok(child) => {
+            let op_name = operation_name.to_string();
+            let mut mqtt_publisher = mqtt_publisher.clone();
+
             tokio::spawn(async move {
                 let logger = log_file.buffer();
-                let _result = child.wait_with_output(logger).await.unwrap();
+
+                // mqtt client publishes executing
+                let topic = C8yTopic::SmartRestResponse.to_topic().unwrap();
+                let executing_str = format!("501,{op_name}");
+                mqtt_publisher
+                    .send(Message::new(&topic, executing_str.as_str()))
+                    .await
+                    .unwrap_or_else(|err| {
+                        error!("Failed to publish a message: {executing_str}. Error: {err}")
+                    });
+
+                // execute the command and wait until it finishes
+                // mqtt client publishes failed or successful depending on the exit code
+                if let Ok(output) = child.wait_with_output(logger).await {
+                    debug!("{output:?}");
```

Comment:
Remove this debug!

Reply:
I would like to keep this debug message. It will show up only if user runs mapper with `--debug`. It's helpful output if user wants to debug if the command was really executed, what was the exit code, stdout and stderr.

Reply:
My motivation to remove this debug log message is that the command status/output/stderr are already logged in a separate file. Logging the path to that file would be more useful.

I agree with @reubenmiller that a log message cannot be dry. One needs some context.

Reply:
Ok, removed.